binancefutures-api/src/main/java/io/contek/invoker/binancefutures/api/websocket/WebSocketMessageParser.java

```diff
@@ -35,6 +40,9 @@ public AnyWebSocketMessage parse(String text) {
     if (obj.has("stream")) {
       return toStreamData(obj);
     }
+    if (obj.has("e")) {
```

Comment:
Given that binance websocket for market data and websocket for user data have absolutely no overlap. Maybe we shall have 2 parsers: UserWebSocketMessageParser, MarketWebSocketMessageParser?

Reply:
This is done.